src/modules/email/email.service.ts

```diff
@@ -0,0 +1,59 @@
+// src/email/email.service.ts
+import { Injectable } from '@nestjs/common';
+import { MailerService } from '@nestjs-modules/mailer';
+import { Article } from './article.interface';
+
+@Injectable()
+export class EmailService {
+  constructor(private readonly mailerService: MailerService) {}
+
+  async sendUserConfirmation(email: string, url: string, token: string) {
```

Comment:
Make this more descriptive like sendUserConfirmationMail, and do the same to others as well